src/test/java/uk/gov/hmcts/reform/sscs/service/RegionalProcessingCenterServiceTest.java

```diff
@@ -119,7 +119,7 @@ public void shouldReturnRegionalProcessingCenterForGivenAppealReferenceNumber()
         assertEquals("0300 123 1142", regionalProcessingCenter.getPhoneNumber());
         assertEquals("0870 324 0109", regionalProcessingCenter.getFaxNumber());
         assertEquals("Liverpool_SYA_Resp@justice.gov.uk", regionalProcessingCenter.getEmail());
-        assertEquals("GAPS", regionalProcessingCenter.getHearingRoute().toString().toUpperCase());
+        assertEquals("gaps", regionalProcessingCenter.getHearingRoute().toString());
```

Comment:
Would assertTrue("gaps").equalsIgnoreCase(regionalProcessingCenter.getHearingRoute().toString()) be better?